site/src/components/WorkspaceStats/WorkspaceStats.tsx

```diff
@@ -79,7 +81,7 @@ const useStyles = makeStyles((theme) => ({
     alignItems: "center",
     color: theme.palette.text.secondary,
     fontFamily: MONOSPACE_FONT_FAMILY,
-    border: `1px solid ${theme.palette.divider}`,
+    margin: "0px",
```

Comment:
I'm _pretty sure_ you can just pass the number `0` like `margin: 0`. No idea if that's preferred or not, just thought I'd mention it.

Reply:
Yeah, you're right. I always add them because I used to get NITed at an old job when I left them off.  A brief google search doesn't reveal any strong opinions. I'm open to any convention!

Reply:
Yea I have no strong feelings either, I really don't know what's best hence curiosity.

I never want to unintentionally nit though, so I try to use conventional comments. I should've said

**Thought(curiosity)**

because code is like a beautiful article that creates interesting discussion, sometimes of which leads to edits, but often just leads to better understandings for future articles.

IDK.

It's friday.